lib/chef/exceptions.rb

```diff
@@ -66,6 +76,9 @@ class User < RuntimeError; end
     class Group < RuntimeError; end
     class Link < RuntimeError; end
     class Mount < RuntimeError; end
+    class Reboot < RuntimeError; end
+    class RebootPending < RuntimeError; end
```

Comment:
Reboot and RebootPending aren't really failures are they? Subclass from Exception instead of RuntimeError so if you're catching RuntimeError for a failure you wouldn't catch these?